tests/zenko_tests/node_tests/backbeat/LifecycleUtility.js

```diff
@@ -349,6 +356,62 @@ class LifecycleUtility extends ReplicationUtility {
             return cb(null, res.Uploads.length > 0);
         });
     }
+
+    /**
+     * Put an object restore request
+     *
+     * @param {string} [versionId] - version id of an object
+     * @param {function} cb - callback function
+     *
+     * @returns {undefined} undefined
+     */
+    putRestoreObject(versionId, cb) {
+        const params = {
+            Bucket: this.bucket,
+            Key: this.key,
+            RestoreRequest: {
+                Days: 1,
+                Tier: 'Standard',
+            },
+        };
+        if (versionId) {
+            params.VersionId = versionId;
+        }
+        this.s3.restoreObject(params, cb);
+    }
+
+    /**
+     * Wait until an archived object is restored
+     *
+     * @param {string} [versionId] - version id of an object
+     * @param {function} cb - callback function
+     *
+     * @returns {undefined} undefined
+     */
+    waitUntilRestored(versionId, cb) {
+        let shouldContinue;
+        const params = {
+            Bucket: this.bucket,
+            Key: this.key,
+        };
+        if (versionId) {
+            params.VersionId = versionId;
+        }
+        return async.doWhilst(
+            next => this.s3.headObject(params, (err, data) => {
+                if (err) {
+                    return next(err);
+                }
+                shouldContinue = !data.StorageClass;
```

Comment:
Also, is the check right? Should it not be `shouldContinue = data.StorageClass;`
Meaning while StorageClass is there, we keep waiting.

Reply:
Bad merge. Corrected now.